backend/package.json

```diff
@@ -1,104 +1,95 @@
 {
-	"name": "codepair-backend",
-	"version": "0.1.14",
```

Comment:
It seems that we should add `version` field.

Reply:
okay !